resources/sass/base/_global.scss

```diff
@@ -12,17 +12,35 @@ html {
   max-width: none;
 }
 
+.rounded-50 {
+  border-radius: 50%;
+}
+
 .max-h-0 {
   max-height: 0;
 }
 
+.h-68 {
```

Comment:
if the class is h-68 does that mean it should be 68 in value? It seems odd the rem value is 24/25. Is it possible to change the class name in use instead?

Reply:
I have updated the value from '-68' to '-96', due to following Tailwind variable naming convention.
 
They seem to be following the same tendency through their set values, where the difference, between size in pixels and numeric variable, is equal to 4. For example, in this case where `96 = 24rem = 384px` => `386px / 96 = 4` => `24rem * 4 = 96`. I hope this makes sense